### DISCUSSION TITLE
You mfs should really check out bypassi's blog for all you complain...

### DISCUSSION THREAD

**Discussion by Huntlb** [3kh0/ext-remover#355] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
blog.bypassi.com

**Comment by Aka-but-nice** [comment]
All read all his blogs, thats how I found out about the exploit at #347

**Comment by 3kh0** [comment]
That is cool, but who asked

**Comment by rwg100** [comment]
i stuck my penith in the car door AHHHHHHHHHH

**Reply by 3kh0** [reply]
how did you do that, it is barely a inch

**Reply by Aka-but-nice** [reply]
Bro got did wrong

**Reply by NathanielThaBeast** [reply]
3kho how u know u been suckin it or sum?????

**Reply by Aka-but-nice** [reply]
3kh0 How you know, huh how you know?

**Comment by Aka-but-nice** [comment]
Also, I realized the exploit at #348 doesn't work.

**Comment by Aka-but-nice** [comment]
It's only for securly. I'll provide an update when a new one comes out. Currently, I'm working on an ingot-based UI for extension removal for v106 or higher.

**Reply by Aka-but-nice** [reply]
Nevermind the exploit at #348 is fixed.
